### DISCUSSION TITLE
Adguard VPN?

### DISCUSSION THREAD

**Discussion by CaptainComeback** [AdguardTeam/AdGuardHome#5603] [Ideas]
Thank you for creating this free for the world to use and I think the only thing missing is the ability to connect to adguard VPN or other VPN services. I know you guys are super busy and keep up the good work.

**Comment by ainar-g** [comment]
Thanks for the kind words!  AdGuard Home is a DNS-level blocker, so it cannot really force its clients to use a VPN.  There are plans to add proxy functionality, but they are very far away, and a lot of other changes need to happen before that.
